README.md

```diff
@@ -18,29 +18,56 @@ This will also install some Git pre-commit hooks that check any modified files a
 
 ### Regular development
 
+#### Dev Entry point
+
+To run the development entry point:
+```sh
+npm run dev
+```
+You can visit the dev entry point at <http://localhost:3000/>.
+Changes to the source files will be applied automatically,
+so you can start this once and then leave it running in the background.
+
+For instructions to see the form in the context of the special page,
+see the [WikibaseLexeme readme](https://gerrit.wikimedia.org/g/mediawiki/extensions/WikibaseLexeme/+/master/README.md).
+
 To run all tests and linters:
 ```sh
 npm t
 ```
 
-Several npm scripts are available to only run a subset of these, such as:
+#### Linting
+The linters, except for TypeScript type checking, can be run with:
 ```sh
-npm run test-only # only tests without linters
-npm run test:unit # only unit tests
+npm run lint
 ```
 
 Many linter errors can be fixed automatically:
 ```sh
 npm run fix
 ```
 
-To run the development entry point:
+Static type checking is executed with
 ```sh
-npm run dev
+npm run check-types
 ```
-You can visit the dev entry point at <http://localhost:3000/>.
-Changes to the source files will be applied automatically,
-so you can start this once and then leave it running in the background.
 
-For instructions to see the form in the context of the special page,
-see the [WikibaseLexeme readme](https://gerrit.wikimedia.org/g/mediawiki/extensions/WikibaseLexeme/+/master/README.md).
+#### Testing
+
+Several npm scripts are available to only run a subset of these, such as:
```

Comment:
“Of these” should be rephrased now that it’s no longer right after the `npm t` command.

Reply:
true, will try to find a better way to say it